aarone_frontend/src/index.html

```diff
@@ -16,8 +16,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBjirg3UoMD5oUiFuZt3P9sErZD-2Rxc68"></script>
 
-  <link rel="icon" type="image/x-icon" href="assets/images/favicon.ico">
-  <link rel="shortcut icon" type="image/x-icon" href="assets/images/favicon.png" />
+  <link rel="icon" type="image/x-icon" href="assets/images/background/logo_4.png">
```

Comment:
[nit] Why didn't you use `favicon.ico` as the file name?

Reply:
fixed, thank you!